docs/source/dev_guide/project-standards.rst

```diff
@@ -152,12 +152,26 @@ Run a tool
 Continuous Integration / Continuous Delivery (CI/CD)
 ----------------------------------------------------
 
-This project uses `GitHub Actions <https://github.com/E3SM-Project/zstash/actions>`_ to run the CI/CD build. The build is triggered by Git ``pull_request`` and ``push`` (merging PRs) events to the the main repo's ``master``.
+This project uses `GitHub Actions <https://github.com/E3SM-Project/zstash/actions>`_ to run two CI/CD workflows.
 
-The CI/CD workflow has two jobs:
+1. CI/CD Workflow
```

Comment:
Is there another name we can come up with so the two workflows aren't "workflow" and "release workflow"? (e.g., this one should have an adjective too).